# Requests an agent to plan the next action
# define the goal; GoalID, GoalStatus are from actionlib_msgs
actionlib_msgs/GoalID goal_id
---
# define the result;
# Note that the action is not returned here because there actions could be of
# different types. Instead, the agent should publish most recent action.
actionlib_msgs/GoalStatus status
---
# no feedback (because planning done through pomdp_py)